<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QfeMainWindowClass</class>
 <widget class="QMainWindow" name="QfeMainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1418</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QfeMainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1418</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSimulation">
    <property name="title">
     <string>Simulation</string>
    </property>
    <addaction name="actionStart"/>
    <addaction name="actionStop"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <widget class="QMenu" name="menuCreate">
     <property name="title">
      <string>Add</string>
     </property>
     <addaction name="actionInspector"/>
     <addaction name="actionTreeView"/>
    </widget>
    <addaction name="menuCreate"/>
   </widget>
   <widget class="QMenu" name="menuAdd">
    <property name="title">
     <string>Add</string>
    </property>
    <widget class="QMenu" name="menuObject">
     <property name="title">
      <string>Object</string>
     </property>
     <addaction name="actionAddCube"/>
     <addaction name="actionAddSphere"/>
    </widget>
    <addaction name="actionAddEmptyObject"/>
    <addaction name="menuObject"/>
   </widget>
   <addaction name="menuSimulation"/>
   <addaction name="menuWindow"/>
   <addaction name="menuAdd"/>
  </widget>
  <widget class="QfeTreeViewDock" name="TreeViewDock">
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string extracomment="TreeView">Project View</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QfeEditorTree" name="EngineTree">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QfeInspectorDock" name="InspectorDock">
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllDockWidgetAreas</set>
   </property>
   <property name="windowTitle">
    <string>Inspector</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QfeEditorInspector" name="Inspector" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="RenderDock">
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Scene</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QWidget" name="RenderWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionStart">
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionInspector">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Inspector</string>
   </property>
  </action>
  <action name="actionTreeView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>TreeView</string>
   </property>
  </action>
  <action name="actionAddEmptyObject">
   <property name="text">
    <string>EmptyObject</string>
   </property>
  </action>
  <action name="actionAddCube">
   <property name="text">
    <string>Cube</string>
   </property>
  </action>
  <action name="actionAddSphere">
   <property name="text">
    <string>Sphere</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QfeEditorTree</class>
   <extends>QTreeWidget</extends>
   <header>qfeeditortree.h</header>
  </customwidget>
  <customwidget>
   <class>QfeEditorInspector</class>
   <extends>QWidget</extends>
   <header>qfeeditorinspector.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QfeInspectorDock</class>
   <extends>QDockWidget</extends>
   <header>qfeinspectordock.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QfeTreeViewDock</class>
   <extends>QDockWidget</extends>
   <header>qfetreeviewdock.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="QfeMainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
